lib/evaluate/character.h

```diff
@@ -70,6 +70,15 @@ template<int KIND> class CharacterUtils {
     return str;
   }
 
+  static Character Pad(const Character &str, std::size_t length) {
```

Comment:
It's not obvious from the `length` variable name if it refers to padding length or new `str` length.
`newLength` might make it clear.

Reply:
Thanks, that is indeed ambiguous. I went for `newLength`. Also, I changed the function name to `Resize` so that it can trim if the `newLength` is smaller.